src/main/java/org/opensearch/performanceanalyzer/collectors/SearchBackPressureStatsCollector.java

```diff
@@ -0,0 +1,742 @@
+        public ResourceUsageTrackerStats(
+                @JsonProperty(
+                                SearchBackPressureStatsValue.Constants
+                                        .SEARCHBP_RESOURCE_USAGE_TRACKER_STATS_CANCELLATIONCOUNT)
+                        long cancellationCount,
+                @JsonProperty(
+                                SearchBackPressureStatsValue.Constants
+                                        .SEARCHBP_RESOURCE_USAGE_TRACKER_STATS_CURRENTMAX)
+                        long currentMax,
+                @JsonProperty(
+                                SearchBackPressureStatsValue.Constants
+                                        .SEARCHBP_RESOURCE_USAGE_TRACKER_STATS_CURRENTAVG)
+                        long currentAvg,
+                @JsonProperty(
+                                SearchBackPressureStatsValue.Constants
+                                        .SEARCHBP_RESOURCE_USAGE_TRACKER_STATS_ROLLINGAVG)
+                        long rollingAvg,
+                @JsonProperty(
+                                SearchBackPressureStatsValue.Constants
+                                        .SEARCHBP_RESOURCE_USAGE_TRACKER_STATS_FRAGMENT)
+                        boolean fragment) {
+            this.cancellationCount = cancellationCount;
+            this.currentMax = currentMax;
+            this.currentAvg = currentAvg;
+            this.rollingAvg = rollingAvg;
+            this.fragment = fragment;
+        }
+        // Getters and Setters
+        public long getCancellationCount() {
+            return cancellationCount;
+        }
+
+        public void setCancellationCount(long cancellationCount) {
+            this.cancellationCount = cancellationCount;
+        }
+
+        public long getCurrentMax() {
+            return currentMax;
+        }
+
+        public void setCurrentMax(long currentMax) {
+            this.currentMax = currentMax;
+        }
+
+        public long getCurrentAvg() {
+            return currentAvg;
+        }
+
+        public void setCurrentAvg(long currentAvg) {
+            this.currentAvg = currentAvg;
+        }
+
+        public long getRollingAvg() {
+            return rollingAvg;
+        }
+
+        public void setRollingAvg(long rollingAvg) {
+            this.rollingAvg = rollingAvg;
+        }
+
+        public boolean isFragment() {
+            return fragment;
+        }
+
+        public void setFragment(boolean fragment) {
+            this.fragment = fragment;
+        }
+    }
+
+    // SearchBackPressureMetrics()
+    /*
+     * SearchBackPressureMetrics(
+     *  SearchTaskStats,
+     *  mode,
+     *  SearchShardTaskStats
+     * )
+     */
+    // Flatten the data fields for easier access
+    public static class SearchBackPressureMetrics extends MetricStatus {
+        private SearchShardTaskStats searchShardTaskStats;
+        private SearchTaskStats searchTaskStats;
+
+        // private double searchBackPressureStatsTest;
```

Comment:
Is this `searchBackPressureStatsTest ` for testing?

Reply:
`// private double searchBackPressureStatsTest;` is not needed for the collector.
thank you so much for pointing out, just removed this piece of code (comment).